tests/utils/exec/tf-exec.go

```diff
@@ -4,17 +4,53 @@ import (
 	"bytes"
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
 	"os/exec"
 	"path"
 
+	con "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
 	h "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/helper"
 	. "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/log"
 )
 
 // ************************ TF CMD***********************************
 
+type ImportArgs struct {
+	URL          string `json:"url,omitempty"`
+	Token        string `json:"token,omitempty"`
+	ResourceKind string `json:"resource_kind,omitempty"`
+	ResourceName string `json:"resource_name,omitempty"`
+	ClusterID    string `json:"cluster_id,omitempty"`
+	ObjectName   string `json:"obj_name,omitempty"`
+}
+
+type ImportService struct {
```

Comment:
as it is separate service - I would put it to separate file

Reply:
fixed